jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -156,6 +156,70 @@ class GemUtils {
         extractGems(project,jRubyClasspath,project.files(gemConfig.files),destDir,action)
     }
 
+    static void setupJars(Configuration config,
+                          File destDir,
+                          GemUtils.OverwriteAction overwrite) {
+        File jarsDir = new File(destDir, 'jars')
+        def artifacts = config.resolvedConfiguration.resolvedArtifacts
+        def fileRenameMap = [:]
+        def coordinates = []
+        def files = []
+        artifacts.each { dep ->
+
+            def group = dep.moduleVersion.id.group
+            def groupAsPath = group.replace('.' as char, File.separatorChar)
+            def version = dep.moduleVersion.id.version
+            // TODO classifier
+            def newFileName = "${groupAsPath}/${dep.name}/${version}/${dep.name}-${version}.${dep.type}"
+
+            if (group != 'rubygems' && dep.type != 'gem' && dep.name != 'jruby-complete') {
```

Comment:
This conditional could use some logic explaining these special cases that we're ignoring for coordinate computation